.github/workflows/release-choco-package.yaml

```diff
@@ -0,0 +1,33 @@
+name: Release - Choco Package
+
+on:
+  release:
+    types:
+      - released
```

Comment:
This will only run when a full official release is done, not a pre-release, right?

Edit: That does appear to be the case according to https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release

Reply:
Right once the full official release is there we will update the TCE version for choco not for the pre-release as in background choco is installing the latest official release.